pkg/domain/entities/container_ps.go

```diff
@@ -46,13 +50,69 @@ type ListContainer struct {
 	// boolean to be set
 	PodName string
 	// Port mappings
-	Ports []ocicni.PortMapping
+	PortMappings []ocicni.PortMapping
 	// Size of the container rootfs.  Requires the size boolean to be true
-	Size *shared.ContainerSize
+	ContainerSize *shared.ContainerSize
 	// Time when container started
 	StartedAt int64
 	// State of container
-	State string
+	ContainerState string
+}
+
+// State returns the container state in human duration
+func (l ListContainer) State() string {
+	var state string
+	switch l.ContainerState {
+	case "running":
+		t := units.HumanDuration(time.Since(time.Unix(l.StartedAt, 0)))
+		state = "Up " + t + " ago"
+	case "configured":
+		state = "Created"
+	case "exited":
+		t := units.HumanDuration(time.Since(time.Unix(l.ExitedAt, 0)))
+		state = fmt.Sprintf("Exited (%d) %s ago", l.ExitCode, t)
+	default:
+		state = l.ContainerState
+	}
+	return state
+}
+
+// Command returns the container command in string format
+func (l ListContainer) Command() string {
+	return strings.Join(l.Cmd, " ")
+}
+
+// Size returns the rootfs and virtual sizes in human duration in
+// and output form (string) suitable for ps
+func (l ListContainer) Size() string {
+	virt := units.HumanSizeWithPrecision(float64(l.ContainerSize.RootFsSize), 3)
+	s := units.HumanSizeWithPrecision(float64(l.ContainerSize.RwSize), 3)
+	return fmt.Sprintf("%s (virtual %s)", s, virt)
+}
+
+// Names returns the container name in string format
+func (l ListContainer) Names() string {
+	return l.ContainerNames[0]
+}
+
+// Ports converts from Portmappings to the string form
+// required by ps
+func (l ListContainer) Ports() string {
+	if len(l.PortMappings) < 1 {
+		return ""
+	}
+	return portsToString(l.PortMappings)
+}
+
+// CreatedAt returns the container creation time in string format.  podman
+// and docker both return a timestamped value for createdat
+func (l ListContainer) CreatedAt() string {
+	return time.Unix(l.Created, 0).String()
+}
+
+// CreateHuman allows us to output the created time in human readable format
+func (l ListContainer) CreatedHuman() string {
```

Comment:
Should` Created() `be human and `CreatedUnix() `otherwise?  I believe `.Create` is what people would expect yes?

Reply:
.CreatedAt is the template key for podmanv1 and docker ... it is a timestamp not human readable.  I added the created myself thinking it might be useful